applications/GeoMechanicsApplication/custom_constitutive/linear_elastic_2D_beam_law.h

```diff
@@ -52,7 +52,7 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) LinearElastic2DBeamLaw : public GeoI
     /// The base class ConstitutiveLaw type definition
     using CLBaseType = ConstitutiveLaw;
 
-    /// The base class ElasticIsotropicK03DLaw type definition
+    /// The base class GeoIncrementalLinearElasticLaw type definition
```

Comment:
I don't think this comment adds anything to what the code already expresses. Since it may get out-of-sync in the future, I would suggest to remove it.